persistence/src/main/java/edu/unc/lib/dl/persist/services/move/MoveObjectsService.java

```diff
@@ -79,7 +81,7 @@ public String moveObjects(AgentPrincipals agent, PID destinationPid, List<PID> p
         job.setOperationMetrics(operationMetrics);
 
         if (asynchronous) {
-            log.info("User {} is queuing move operation {} of {} objects to destination {}",
+            log.info("User {} is queueing move operation {} of {} objects to destination {}",
```

Comment:
I'm not claiming to be a spectacular speller, but just saying: https://english.stackexchange.com/questions/60852/queueing-or-queuing :)